src/components/Link/Object.tsx

```diff
@@ -170,15 +176,15 @@ export const LinkContact = () => {
                             );
                           }
                         }}
-                      ></Checkbox>
+                      />
                     </Stack>
                     <Stack style={{ width: "92%" }}>
                       <FieldMapping
                         items={[issue]}
                         hasCheckbox={true}
                         metadata={usableFields}
                         externalChildUrl={IssueJson.externalChildUrl}
-                        childTitleAccessor={(e) => e[IssueJson.titleKeyName]}
+                        childTitleAccessor={(e) => e[IssueJson.titleKeyName] as string}
```

Comment:
Why do we type it a string, I assume the linter thinks it can be something else. How is it proven that it wont be something else?